mcs/packages/Makefile

```diff
@@ -19,11 +19,8 @@ ROSLYN_FILES_FOR_MONO = \
 	$(ROSLYN_CSC_DIR)/Microsoft.CodeAnalysis.dll		\
 	$(ROSLYN_CSC_DIR)/Microsoft.CodeAnalysis.Scripting.dll \
 	$(ROSLYN_CSC_DIR)/System.Collections.Immutable.dll	\
-	$(ROSLYN_CSC_DIR)/System.Numerics.Vectors.dll		\
 	$(ROSLYN_CSC_DIR)/System.Reflection.Metadata.dll	\
 	$(ROSLYN_CSC_DIR)/System.Runtime.CompilerServices.Unsafe.dll	\
-	$(ROSLYN_CSC_DIR)/System.Threading.Tasks.Extensions.dll	\
```

Comment:
These two are mobile profiles only and we'll have to copy them from somewhere

Reply:
So we do want to include them in the roslyn repo and copy them in? Or only copy them for mobile profiles? I'm not clear on what this means

Reply:
We'll have to copy them from roslyn

Reply:
OK. Should we do this for all profiles?

Reply:
yes (we install roslyn for net_4_x only)